airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_message.yaml

```diff
@@ -118,10 +118,43 @@ definitions:
       name:
         type: string
         description: Stream's name.
-      json_schema:
-        description: Stream schema using Json Schema specs.
+      discovered:
```

Comment:
catalog or structure?

Reply:
haha. don't want to use the word catalog in a stream that's in a catalog. 😅 . i'm okay with structure. not in love with it.

Reply:
what about `info`?

Reply:
This is actually the `schema`.

Reply:
well it contains the schema (`json_schema` field). if i recall there's a good reason that we can't use the word `schema` as the key.

Reply:
`stream_schema`?

Reply:
moot with new structure.